servers/node-solid-server/.db/oidc/op/provider.json

```diff
@@ -0,0 +1,411 @@
+{
+  "issuer": "https://localhost:8443",
+  "authorization_endpoint": "https://localhost:8443/authorize",
+  "token_endpoint": "https://localhost:8443/token",
+  "userinfo_endpoint": "https://localhost:8443/userinfo",
+  "jwks_uri": "https://localhost:8443/jwks",
+  "registration_endpoint": "https://localhost:8443/register",
+  "response_types_supported": [
+    "code",
+    "code token",
+    "code id_token",
+    "id_token",
+    "id_token token",
+    "code id_token token",
+    "none"
+  ],
+  "response_modes_supported": [
+    "query",
+    "fragment"
+  ],
+  "grant_types_supported": [
+    "authorization_code",
+    "implicit",
+    "refresh_token",
+    "client_credentials"
+  ],
+  "subject_types_supported": [
+    "public"
+  ],
+  "id_token_signing_alg_values_supported": [
+    "RS256",
+    "RS384",
+    "RS512",
+    "none"
```

Comment:
I haven't seen the code, but the existence of the "none" algorithm in JWT is generally seen as a security blunder. So wouldn't it be better to explicitly not support it in the test suite?

Reply:
Good point! This is in the default config of node-solid-server.
I'm not sure yet which algorithms a server should support as a minimum, but the test suite should test that "none" is not supported (the DPop spec mentions that).